prow/getting_started_deploy.md

```diff
@@ -8,21 +8,23 @@ Both of these are focused on [Kubernetes Engine](https://cloud.google.com/kubern
 
 ## Tackle deployment
 
-Prow's tackle utility walks you through deploying a new instance of prow in a couple minutes, try it out!
+Prow's `tackle` utility walks you through deploying a new instance of prow in a couple minutes, try it out!
 
-You need three things:
-1) The prow tackle command.
-2) a [personal access token][1] for the github bot user you want to act as prow
+You need a few things:
 
+1. [`bazel`](https://bazel.build/) build tool installed and working
+1. The prow `tackle` utility. It is recommended to use it by running `bazel run //prow/cmd/takle` from `test-infra` directory, alternatively you can install it by running `go get -u k8s.io/test-infra/prow/cmd/tackle` (in that case you would also need go installed and working).
+1. A [personal access token][1] for the github bot user you want to act as prow. If you're setting up prow for personal repo (not organization) then bot user == you (your github account).
```

Comment:
`github` -> `GitHub`

Reply:
> If you're setting up prow for personal repo (not organization) then bot user == you (your github account).

but you can still setup a separate account to act as the prow user on a person repo.
i would omit this second sentence.

Reply:
I actually wasn't able to do that. I created separate github user, generated personal access token. From personal account invited bot user as collaborator. Then when i ran `tackle` and specified bot's personal access token I've got following error:
```
yb172/experiments
INFO[0029] ListRepoHooks(yb172, experiments)             client=github
FATA[0036] Could not configure repos to send http://<ip>/hook webhooks.  error="find http://<ip>/hook hook in yb172/experiments: list hooks: return code not 2XX: 404 Not Found"
```
[Line where failure happens](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tackle/main.go#L598)
There is a call to get repo hooks from github and it returns 404. My guess is that's because bot user was invited as "collaborator" and it doesn't have permissions to read and write repository hooks. Two things:
 * I've tried to add "repo hooks" permissions to bot's personal access token - same error
 * (Using my account) In repository settings I don't have any control over permissions that I give to collaborators
<img width="741" alt="screen shot 2019-03-03 at 10 38 02 am" src="https://user-images.githubusercontent.com/550350/53699747-8a8bf300-3da0-11e9-8755-50daf42b8c1a.png">

I added this note because it didn't work for me. Please let me know if I'm doing something wrong and I'll try to get it working but otherwise this note would have saved me some time setting up prow.